src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7709,6 +7709,7 @@ public enum NormalizationForm
         [CLSCompliant(false)]
         public static bool TryCreate(uint value, out Rune result) { throw null; }
         public bool TryEncode(Span<char> destination, out int charsWritten) { throw null; }
+        public bool TryEncodeToUtf8Bytes(Span<byte> destination, out int bytesWritten) { throw null; }
```

Comment:
Is this a approved public API that we are comfortable with?

Reply:
I believe so, it was discussed today.  @GrabYourPitchforks what was the final word?

Reply:
It's scheduled to go through review, still: https://github.com/dotnet/corefx/issues/35530